src/app/shared/utils/utils.ts

```diff
@@ -43,6 +50,22 @@ export class Utils {
     return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
   }
 
+  /**
+   * Truncates text to specified length and then adds ellipes.
+   *
+   * @param stringToTruncate The string to truncate.
+   * @param charsToTruncateBy The characters of the string to keep.
+   * @returns The truncated string plus ellipses(three periods).
+   */
+  public truncateText( stringToTruncate: string, charsToTruncateBy: number ): string {
```

Comment:
not sure why we are creating our own functions for this.. I'm sure we are using Lodash
https://lodash.com/docs/4.17.15#truncate

Reply:
Thanks for pointing that out. I didn't know lodash had that.